pages/Home.tsx

```diff
@@ -0,0 +1,12 @@
+import React, { useEffect } from "react"
+import BundlesOverview from "../components/bundles/BundlesOverview"
+import * as ga from "../lib/ga"
+
+export default function Home() {
+  // eslint-disable-next-line react-hooks/exhaustive-deps
+  useEffect(ga.pageview)
+  // TODO: Move query logic to
```

Comment:
where to? 😅

Reply:
Not entirely sure what the intention was with the analytics functions as it's not making use of the hook correctly